frontend/src/pages/cellguide-sitemap.xml/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { GetServerSideProps } from "next";
+import { getServerSideSitemap, ISitemapField } from "next-sitemap";
+import { convertTimestamp } from "src/common/sitemaps/utils";
+import { CELLGUIDE_DATA_URL } from "src/configs/configs";
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const latestSnapshotRes = await fetch(
+    `${CELLGUIDE_DATA_URL}/latest_snapshot_identifier`
+  );
+
+  const latestSnapshotId = await latestSnapshotRes.json();
+
+  const response = await fetch(
+    `${CELLGUIDE_DATA_URL}/${latestSnapshotId}/celltype_metadata.json`
```

Comment:
Can we also add tissues to the sitemap?

Tissues can be found in:
`${CELLGUIDE_DATA_URL}/${latestSnapshotId}/tissue_metadata.json`

Tissue pages can be accessed with:
`https://cellxgene.cziscience.com/cellguide/tissues/{TISSUE_ID}` (e.g. TISSUE_ID could be "UBERON_0002048")

Reply:
Added!